app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/CellPlotServiceTest.java

```diff
@@ -129,24 +130,40 @@ void getCellPlotParameter() {
     }
 
     @Test
-    void fetchDefaultPlotMethodWithParameterisation(){
+    void fetchDefaultPlotMethodWithParameterisation() {
         when(cellPlotDaoMock.fetchDefaultPlotMethodWithParameterisation("E-CURD-4"))
-                .thenReturn(ImmutableMap.of("umap",
+                .thenReturn(ImmutableMap.of("UMAP",
```

Comment:
I think the last time we discussed that we are going to generate a random string as the name of the plot type and accession.
I also think that you have the helper method for that. Could you please use that method here, too?
Thanks